### PR TITLE
cd ansible-release: include artifact URL in PR body

### PR DESCRIPTION
**Pull request by gotmax23**
This exposes the release artifact download URL in the PR body so we can access it while reviewing the release PR.

### CODE REVIEW

**Comment by gotmax23** [comment]
Submitted test job https://github.com/ansible-community/ansible-build-data/actions/runs/8069875373

File: .github/workflows/ansible-release.yml [hunk 1]
```diff
@@ -93,12 +94,14 @@ jobs:
...
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: >-
-          gh pr create
-          --base main
-          --head "publish-${ANSIBLE_VERSION}"
-          --title "Release Ansible ${ANSIBLE_VERSION}"
-          --body "${CI_COMMIT_MESSAGE}"
+          ARTIFACT_URL: ${{ steps.upload-artifact.outputs.artifact-url }}
+        run: |
+          body="$(echo "${CI_COMMIT_MESSAGE}\nRelease artifacts: <${ARTIFACT_URL}>")"
```
[the remark this answers is not in the document]
Comment (felixfontein) [reply .github/workflows/ansible-release.yml:99]: You need to pass `-e` to `echo` to interpret escape sequences:
```suggestion
          body="$(echo -e "${CI_COMMIT_MESSAGE}\nRelease artifacts: <${ARTIFACT_URL}>")"
```

**Comment by gotmax23** [comment]
Let's try this again: https://github.com/ansible-community/ansible-build-data/actions/runs/8070225169

File: .github/workflows/ansible-release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (gotmax23) [reply .github/workflows/ansible-release.yml]: Thanks for the fix!

**Comment by gotmax23** [comment]
Ready for review

**Comment by webknjaz** [comment]
@gotmax23 could you also put the PR and artifact URLs into the job summary?

**Comment by gotmax23** [comment]
The artifacts already show up in the summary but only _after_ the entire workflow has finished. This way, the link is available during the PR review phase.

**Comment by webknjaz** [comment]
@gotmax23 totally. I'm just asking for the job summary output additionally. This way, it's possible to see this information on that page too. Without having to go to the PR and then to the artifact. I don't think it's harmful to have the links in two places.

**Comment by gotmax23** [comment]
I guess I don't know exactly what you mean. Are you referring to, for example, https://github.com/ansible-community/ansible-build-data/actions/runs/8067689405 or is there some other GHA "summary" feature that I don't know about?

**Comment by webknjaz** [comment]
Random repo example: https://github.com/pyOpenSci/pyosMeta/actions/runs/8058291379#summary-22010849191 — this is a place where you can output markdown attached to workflow runs. Each job can write there and would get its own widget.

https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/

**Comment by gotmax23** [comment]
I'll merge this for today's release. We can handle the summary part separately.

**Pull request merged**